runtime/devnet/src/config/ismp.rs

```diff
@@ -0,0 +1,58 @@
+use frame_support::traits::Get;
+use frame_system::EnsureRoot;
+use ismp::{error::Error, host::StateMachine, module::IsmpModule, router::IsmpRouter};
+use ismp_parachain::ParachainConsensusClient;
+use sp_std::prelude::*;
+
+use crate::{
+	AccountId, Balance, Balances, Ismp, IsmpParachain, ParachainInfo, Runtime, RuntimeEvent,
+	Timestamp,
+};
+
+impl pallet_ismp::Config for Runtime {
+	type AdminOrigin = EnsureRoot<AccountId>;
+	type Balance = Balance;
+	type ConsensusClients = (ParachainConsensusClient<Runtime, IsmpParachain>,);
+	type Coprocessor = Coprocessor;
+	type Currency = Balances;
+	type HostStateMachine = HostStateMachine;
+	type Mmr = pallet_ismp::NoOpMmrTree<Self>;
```

Comment:
Any reason for this setting?

Reply:
See https://github.com/r0gue-io/ismp/blob/49f8a18d5ea1ade3e6c456a6bbd402c2713c5ed6/modules/ismp/pallets/pallet/src/lib.rs#L122